notebooks/NIRCam/NIRCam_claw_detection/nircam_claw_detection.ipynb

```diff
@@ -0,0 +1,461 @@
+    "supported_observations = program.observations.supported_observations.keys()\n",
+    "for obs_number in supported_observations:\n",
+    "    program.plot_observation(program.observations.data[obs_number])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9d8f4b2d-c969-4e9b-b4ed-e5776184dbf6",
+   "metadata": {},
+   "source": [
+    "<a id='v2-vs-v3'></a>\n",
+    "### Plotting a Single Angle of Attitude in V2 & V3 Space\n",
+    "\n",
+    "The figure below shows what the tool is searching for when sweeping angles. Here for a single angle we show the V2 & V3 space of the targets in our source catalog as well as the susceptibility region. The intensity of targets that fall in this region are dependent on the location where of the target in the susceptibility region and the brightness of the target."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b4b4cda4-c003-4d9d-abf3-5758ba8e517c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from jwst_rogue_path_tool.plotting import plot_fixed_angle_regions\n",
+    "\n",
+    "angles = np.arange(100, 106)\n",
+    "\n",
+    "for angle in angles:\n",
+    "    plot_fixed_angle_regions(observation, angle)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b647f76c-b6c1-42db-a518-9add1195efd4",
+   "metadata": {},
+   "source": [
+    "<a id='attitude-vs-flux'></a>\n",
+    "### Flux as a Function of Attitude\n",
+    "\n",
+    "Below we show the calculated flux as a function of attitude angle. This figure also ties in the [JWST Backgrounds Tool](https://github.com/spacetelescope/jwst_backgrounds) to calculate the background of the target JWST is observering described in the APT file. By default, we set the parameters for the backgrounds tool to calculate 10% of the minimum background value. You can set the function and percentage via a dictionary and pass it to the AptProgram class.\n",
+    "~~~\n",
+    "jbt_params = [{\"threshold\":0.1, function:np.min}, {\"threshold\":0.2, \"function\":np.mean}, ..... {\"threshold\":value, \"function\":stats_function}]\n",
+    "\n",
+    "AptProgram(apt_JSON_filename, angular_step=1.0, bkg_params=jbt_params)\n",
+    "~~~"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "758ff490-ea7c-40e6-9bca-5faa7f9b50e5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from jwst_rogue_path_tool.plotting import create_v3pa_vs_flux_plot\n",
+    "\n",
+    "for obs_number in supported_observations:\n",
+    "    create_v3pa_vs_flux_plot(program.observations.data[obs_number])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eae8dd72-babc-456e-9e6e-231a80192896",
+   "metadata": {},
+   "source": [
+    "<a id='reporting-and-apt'></a>\n",
+    "## Reporting & Applying PA Constraints in APT\n",
+    "\n",
+    "Below, we show the flux plots for observation 5 as well as writing out the report. We also discuss how to apply these constraints to your program."
```

Comment:
Could we change this to: "In this section, we demonstrate how to plot the predicted claw flux across all position angles for a single observation, identify the angles free from claws, and modify the observation in APT to ensure it is taken only within those good position angle ranges."